pkg/data/components/HoneycombExporter.yaml

```diff
@@ -151,7 +132,7 @@ templates:
         value: "{{ .Values.Insecure | encodeAsBool }}"
         suppress_if: "{{ not .Values.Insecure }}"
       - key: "{{ .ComponentName }}.headers.x-honeycomb-team"
-        value: "{{ .Values.APIKey }}"
+        value: "__ENV:{{ .Values.Environment }}__"
```

Comment:
We'll need more complex logic here if we're going to support both automatic key creation and our existing `${HTP_EXPORTER_APIKEY}` solution.